almanach/common/AlmanachException.py

```diff
@@ -12,5 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
```

Comment:
Just saw that the name of the file for exception are not standard : camel case instead of snake case